src/targets/mobile/containers/RatingModal.jsx

```diff
@@ -0,0 +1,178 @@
+/* global cozy AppRate */
+import React, { Component } from 'react'
+import Modal, { ModalContent } from 'cozy-ui/react/Modal'
+import { connect } from 'react-redux'
+import { promptedRating, promptLater } from '../actions/rating'
+import { softwareID } from '../lib/cozy-helper'
+
+import styles from '../styles/rating'
+
+const SCREEN_ENJOY = 'SCREEN_ENJOY'
+const SCREEN_FEEDBACK = 'SCREEN_FEEDBACK'
+
+const BUTTON_INDEX_RATE = 1
+const BUTTON_INDEX_LATER = 2
+
+const FEEDBACK_EMAIL = 'support@cozycloud.cc'
+const PROMPT_AFTER_BOOTS = 5
+
+class EnjoyCozy extends Component {
+  render ({ onReply }) {
+    const { t } = this.context
+    return (
+      <Modal
+        title={t('mobile.rating.enjoy.title')}
+        withCross={false}
+      >
+        <ModalContent>
+          <div className={styles['button-block']}>
+            <button className='coz-btn coz-btn--secondary' onClick={() => onReply(false)}>
+              { t('mobile.rating.enjoy.no') }
+            </button>
+            <button className='coz-btn coz-btn--regular' onClick={() => onReply(true)}>
+              { t('mobile.rating.enjoy.yes') }
+            </button>
+          </div>
+        </ModalContent>
+      </Modal>)
+  }
+}
+
+class FeeedbackForm extends Component {
+  getInitialState () {
+    return {
+      sending: false
+    }
+  }
+
+  async onSubmit (e) {
+    e.preventDefault()
+
+    const { t } = this.context
+    const mailData = {
+      mode: 'from',
+      to: [
+        { name: 'Support', email: FEEDBACK_EMAIL }
+      ],
+      subject: t('mobile.rating.email.subject'),
+      parts: [
+        { type: 'text/plain', body: this.textarea.value.toString() }
+      ]
+    }
+
+    try {
+      this.setState({ sending: true })
+      await cozy.client.jobs.create('sendmail', mailData)
+    } catch (e) {
+      // Sending the email failed; this can happen because of insuficient permissions for example. Not a big deal either in this context.
```

Comment:
The state gets updated just below. We've decided to silently ignore this error as addressing it requires making the user sign in again, which isn't worth it for this feature.